### DISCUSSION TITLE
Ability to shift the file class fields section with mouse

### DISCUSSION THREAD

**Discussion by ghost** [mdelobelle/metadatamenu#512] [Ideas]
Hello, It would be cool if we can change the ordering of the fields by clicking and moving them

**Comment by ghost** [comment]
As of right now, I move them into a parent field and then put them back into the original field in the order I want

**Reply by sector101010** [reply]
I'd like easier reordering as well, but just FYI, if you open a fileClass file in a text editor outside obsidian, you'll see a property `fields` in the frontmatter containing the list of fields. You can re-order them there as well, therefore for me it's not urgent.

**Comment by mdelobelle** [comment]
will be possible in the next version (0.7.1) should be released very soon
